src/skyflow.ts

```diff
@@ -136,6 +136,18 @@ class Skyflow {
 
   static init(config: ISkyflow): Skyflow {
     const logLevel = config?.options?.logLevel || LogLevel.ERROR;
+    if (
```

Comment:
can we extract this piece of code to helper function.

Reply:
can i call that helper function inside the if block or should i move if block to the helper function as well.